ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -489,6 +493,24 @@
         }
     });
 
+    function registerRestEndpointUrlValidator() {
+        $(window).adaptTo("foundation-registry").register("foundation.validation.validator", {
+            selector: "[data-rest-endpoint-url-validation]",
+            validate: (el) => {
+                const url = el.value;
+
+                // Regex to validate absolute URLs starting with http:// or https:// only
+                const absoluteUrlPattern = /^https?:\/\/.+$/i;
+
+                if (!absoluteUrlPattern.test(url)) {
+                    return Granite.I18n.getMessage(
+                        "Enter a valid URL for the POST request. Ensure the URL starts with 'https://' or 'http://' and follows this structure: https://www.server.com/path/to/resource"
```

Comment:
@s1mahanty my review comment for i18n is not resolved. I checked - we need to add the new string to i18n repo, please log a ticket for translation team to incorporate this string.

Reply:
Similar change is needed to update the help text as well - to remove the path browse part of the string.

Reply:
Sure, Sudhanshu but there is no action items in this PR for the translation, as it is auto picked up by the translation team to add in the i18n repo